### PR TITLE
[Method References] LPS-160434 Convert method references to field references in class extends BasePanelApp

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , Based on the comments in https://github.com/liferay-core-infra/liferay-portal/pull/1435, I only did one thing first in this PR, converting method references to field references.

Detial: https://github.com/liferay-core-infra/liferay-portal/pull/1435

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by catarineaguiar** [comment]
@W2jx no regressions found.

**Review by dantewang — APPROVED** [review]
@tinatian

**Comment by dantewang** [comment]
@W2jx Please rebase

**Comment by W2jx** [comment]
Force-push just to resolve the conflict about the order of imported package items in file modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/application/list/ObjectEntriesPanelApp.java

File: modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/application/list/NotificationsPanelApp.java [hunk 1]
```diff
@@ -63,7 +63,6 @@ public String getPortletId() {
 	}
 
 	@Override
-	@Reference(unbind = "-")
 	protected void setUserNotificationEventLocalService(
```
Comment (tinatian) [inline modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/application/list/NotificationsPanelApp.java:66]: @W2jx Why we need to keep setUserNotificationEventLocalService()

**Pull request closed without merging**